package.json

```diff
@@ -1,125 +1,50 @@
 {
-  "name": "blade",
   "private": "true",
   "author": "kamleshchandnani",
   "version": "0.0.1",
   "license": "MIT",
   "engines": {
-    "node": ">=12.19.0"
+    "node": ">=14.15.0"
```

Comment:
Should we do `14.16.0`? LTS release

Reply:
vercel doesn't supports 14.16 yet hence had to roll it back to this version else the build fails on vercel 😿

Reply:
Oh weird, I remember checking this [announcement](https://vercel.com/changelog/node-js-14-lts-is-now-available) sometime back.

Reply:
yes that's why I added 14.16 but it failed saying expected 14.16 but found 14.15 😞